src/background/sidebar-injector.ts

```diff
@@ -223,7 +223,7 @@ export class SidebarInjector {
 
     function isLMSAssignmentURL(url: string) {
       const { origin } = new URL(url);
-      // Matches origins like `lms.hypothes.is`, `qa-lms.hypothes.is`, `lms.ca.hypothes.is`.
+      // Matches origins like `lms.hypothes.is`, `staging-lms.hypothes.is`, `lms.ca.hypothes.is`.
```

Comment:
I think the staging URL for the LMS is `lms.staging.hypothes.is`

The logic would still work.

Reply:
That's actually fine. It is intentional that these are only example origins and might differ if we decide to change the domain name syntax slightly for whatever reason, as long as we still include "lms" somewhere.